changelog/private_member_traits.dd

```diff
@@ -0,0 +1,14 @@
+Enable private member access for traits
+
+The following traits can now access non-public members:
+$(UL
+    $(LI hasMember)
```

Comment:
I think this worked before this PR. Since this is not try to access the symbol. I'm guessing that what's `ignoresymbolvisibility` is for.